src/cli/CleanupDatabasesCommand.ts

```diff
@@ -0,0 +1,37 @@
+import {OfflineDataProcessor} from "../database/OfflineDataProcessor";
+import {DatabaseConnection} from "../database/DatabaseConnection";
+
+
+export class CleanupDatabasesCommand {
+
+  public constructor(
+      private readonly databaseConnection: DatabaseConnection,
+      private readonly offlineProcessor: OfflineDataProcessor
+  ) {
+
+  }
+
+  public async run(argv: string[]): Promise<void> {
+    // Check if new views in original database works fine
+    const tables = this.offlineProcessor.getTablesList();
+
+    const checkAlltables = () => {
```

Comment:
how about `private checkAlltables() {}` instead of closure ?

Reply:
:+1: